unic/ucd/normal/src/gen_cat.rs

```diff
@@ -10,17 +10,28 @@
 // except according to those terms.
 
 
+#[cfg(not(feature = "unic-ucd-category"))]
```

Comment:
These should work, but also we can make it more clear by creating a local `mod` per feature-status, and put the `cfg` gate on the whole `mod`. Would be much easier to read and track relations between `use` and definitions.